pandas/tests/io/sas/test_xport.py

```diff
@@ -157,3 +158,11 @@ def test_truncated_float_support(self):
 
         data = read_sas(self.file04, format="xport")
         tm.assert_frame_equal(data.astype("int64"), data_csv)
+
+    def test_cport_header_found_raises(self):
+        # Test with DEMO_PUF.cpt, the beginning of puf2019_1_fall.xpt
+        # from https://www.cms.gov/files/zip/puf2019.zip
```

Comment:
Is this public domain or similar?  If not, might be good to have someone create a CPT file from scratch.

Reply:
It's publicly available data (from the US government's CMS, stored at https://www.cms.gov/files/zip/puf2019.zip). I don't have SAS so can't create a file. But the compressed CPORT files will have `**COMPRESSED** **COMPRESSED** **COMPRESSED** **COM` as the first 40 characters (as documented [here](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/movefile/p1bm6aqp3fw4uin1hucwh718f6kp.htm)), so I just opened this file in a text editor and deleted everything after the first few lines.